### PR TITLE
Added support for social account authentication

### PR DESCRIPTION
**Pull request by jenia90**
Referencing #17 
So far I've only tested it with google sign-in and it worked like charm. 
I also updated the README with links and instructions on how to generate and add user-secrets for each social provider.

### CODE REVIEW

File: TodoList.Web/Startup.cs [hunk 1]
```diff
@@ -34,6 +34,7 @@ public void ConfigureServices(IServiceCollection services)
             services.ConfigureSecurity();
             services.ConfigureIdentity();
             services.ConfigureStorage(Configuration);
+            services.ConfigureSocialAuthentication(Configuration);
```
Comment (amoraitis) [inline TodoList.Web/Startup.cs]: Please fix the formatting.

**Review by amoraitis — COMMENTED** [review]
Other than the formatting issue, it looks good.
Please update your branch.

File: TodoList.Web/Startup.cs (hunk 1 above)
Comment (jenia90) [reply TodoList.Web/Startup.cs]: You mean change it to `ConfigureAuthentication`?

**Review by amoraitis — APPROVED** [review]
Thanks for this contribution!

**Pull request merged**